automated/linux/armnn/pyarmnn-bench.yaml

```diff
@@ -0,0 +1,25 @@
+metadata:
+    format: Lava-Test Test Definition 1.0
+    name: pyarmnn-benchmark
+    description: "Part of ml-bench, pyarmnn-bench testing validity of pyarmnn pip wheels."
+    maintainer:
+        - theodore.grey@linaro.org
+    os:
+        - debian
+    scope:
+        - functional
+    devices:
+        - synquacer
+        - dragonboard-845c
+        - hi960-hikey
+        - stm32mp1
+params:
+    ARMNN_TARBALL: 'https://snapshots.linaro.org/componenets/armnn/latest/armnn.tar.xz'
```

Comment:
"components"

Reply:
This should be resolved when this patch is merged.

Reply:
https://snapshots.linaro.org/componenets/ is this a valid path ?

Reply:
Is this a typo in yaml file ?

From yaml script,
params:
    ARMNN_TARBALL: 'https://snapshots.linaro.org/componenets/armnn/latest/armnn.tar.xz'

from shell script
ARMNN_TARBALL="https://snapshots.linaro.org/components/armnn/latest/armnn-full.tar.xz"

Reply:
Hi,
Sorry my mistake I misread and thought this was a previously fixed issue in armnn-unit-tests.yaml and not pyarmnn-bench.yaml. 
Should be resolved now, thanks for the help.